# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/metrics.yaml
# Commit: dc509eab3ed7
# Full Hash: dc509eab3ed7a513258723eb46b25f33a610e720
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-07-11 16:36:21
# Regressor Bug: 1953221
# File Overlap Count: 3
# Description:
#   Bug 1953221 - don't accept SCTs with a timestamp after the distrustAfter date for the root r=jschanck
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D256720
# ==============================================================================

diff -r 5b86ae2c81f9 -r dc509eab3ed7 security/manager/ssl/metrics.yaml
--- a/security/manager/ssl/metrics.yaml	Thu Jul 10 18:37:49 2025 +0000
+++ b/security/manager/ssl/metrics.yaml	Thu Jul 10 19:04:10 2025 +0000
@@ -1613,19 +1613,17 @@
     description: >
       Verification status of Signed Certificate Timestamps received (0=Decoding
       error, 1=Valid SCT, 2=SCT from unknown log, 3=Invalid SCT signature, 4=SCT
-      timestamp is in the future, 5=Valid SCT from a disqualified log)
-
-      This metric was generated to correspond to the Legacy Telemetry enumerated
-      histogram SSL_SCTS_VERIFICATION_STATUS.
+      timestamp is in the future, 5=Valid SCT from a disqualified log 6=SCT
+      timestamp is after the root's distrustAfter time).
     range_min: 0
     range_max: 10
     bucket_count: 11
     histogram_type: linear
     bugs:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1293231
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1953221
     data_reviews:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1293231
     notification_emails:
       - seceng-telemetry@mozilla.com
     expires: never
-    telemetry_mirror: SSL_SCTS_VERIFICATION_STATUS